### ISSUE TITLE
Failed notifications won't disappear

### ISSUE THREAD

**Issue by sollidsnake** [franciscolourenco/done#51]
When I have some command that returns a non-zero status, the notification stays on the screen forever, until I manually close it. I'm using done with KDE Plasma, under Arch Linux, and this only happens for notifications of this package.

**Comment by unai-ndz** [comment]
It's probably because they are tagged as "critical". Look if there is a setting under KDE or the program used to display notifications to change the timeout of critical notifications.

**Comment by sollidsnake** [comment]
Indeed there's a setting for the notifications timeout in KDE, which is by default configured to 6 seconds, but it doesn't seem to have setting related to "critical". So you believe this is a KDE issue?

**Comment by unai-ndz** [comment]
The current behavior is defined in the [specification](https://developer.gnome.org/notification-spec/). Look at urgency levels. And failed commands being critical seams sane for me, it's actually what i want. I think it's weird KDE does not have a configuration option for that, as every program for notifications I have used have something like that and KDE usually has a lot of customization afaik. I think an option for customization fits well in both sides tho.

**Comment by sollidsnake** [comment]
Indeed KDE is very customizable, but it doesn't seem to have anything related to critical notifications not disappearing.
These are the options I have on my system settings:
![Screenshot_20190928_163620](https://user-images.githubusercontent.com/1778350/65821610-1e764e00-e20e-11e9-9383-f3e3d425d116.png)
![Screenshot_20190928_164302](https://user-images.githubusercontent.com/1778350/65821676-0bb04900-e20f-11e9-8b60-9820cd1a010e.png)

I tried messing around with them but the notification just won't go away by itself. But you are right, this issue is not related to this package. I tested with `notify-send -u critical` and the same problem happens. I'll ask this on a KDE forum to make sure I'm not missing anything.
